docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,73 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+List all the `tables` from the database with `database-name` and `is temporary table`.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} database_name] [LIKE 'regex_pattern']
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>{FROM|IN} database_name</em></code></dt>
+  <dd>
+     Specifies the `database` name from which tables are listed.
+  </dd>
+  <dt><code><em>LIKE 'regex_pattern'</em></code></dt>
+  <dd>
+     Specifies the regex pattern that is used to filter out unwanted tables.
+       <br> - Only `*` and `|` are allowed as wildcard pattern.
+       <br> - Excluding `*` and `|` the remaining pattern follows the regex semantics.
+       <br> - The leading and trailing blanks are trimmed in the input pattern before processing.  
+  </dd>
+</dl>
+
+### Example
+{% highlight sql %}
+-- List all tables in default database
+SHOW TABLES;
+  +-----------+------------+--------------+--+
```

Comment:
Please lets fix the formatting. Did you construct this output by hand ? Can we paste the
actual output here please so we preserve the formatting.

Reply:
@dilipbiswal 
I just Copy and paste the actual output from the Beeline console
Here is the beeline console snap
![image](https://user-images.githubusercontent.com/7912929/65381823-68df6280-dd16-11e9-873c-c49244e7aee7.png)

In the UI it looks like below, i think it is normal
![image](https://user-images.githubusercontent.com/7912929/65381847-08045a00-dd17-11e9-9d3f-69ec058eaa98.png)

Only in the git file editor, it shows inconsistent.